.github/workflows/performance.yml

```diff
@@ -49,3 +49,37 @@ jobs:
         env:
           SWIFT_SUFFIX: 'Swift'
           SAMPLE: Performance
+      - name: Build and Test SwiftUI iOS
+        run: |
+          set -io pipefail
+          cd performance/SwiftUIPerformanceQuickstart/PerformanceExample
+          bundle exec pod install --repo-update
+          xcodebuild \
+           -workspace "$workspace_path" \
+           -scheme "${SAMPLE}Example${SWIFT_SUFFIX}" \
+           -sdk iphonesimulator \
+           -destination 'platform=iOS Simulator,name=iPhone 11' \
+           build \
+           test \
+           | xcpretty
+        env:
+          SWIFT_SUFFIX: ' (iOS)'
+          SAMPLE: Performance
+          workspace_path: PerformanceExample.xcworkspace
+      - name: Build and Test SwiftUI tvOS
```

Comment:
Could we use a [matrix](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) here for ios and tvos

Reply:
I'm not sure how to incorporate it since `scheme`, `sdk`, and `destination` only correspond with either iOS or tvOS and shouldn't be mixed and matched...

Reply:
Yep, I think to apply `matrix`, we might need a script to handle different params/flags. We can leave it like that before we have a script for all tests.